SVS/model/utils/SVSDataset.py

```diff
@@ -60,11 +60,13 @@ def _get_spectrograms(
     # mag = np.abs(linear)  # (1+n_fft//2, T)
 
     if n_mels > 0:
-        mel_basis = librosa.filters.mel(require_sr, n_fft, n_mels)
-        mel = np.dot(mel_basis, mag)  # (n_mels, t)
-        mel = 20 * np.log10(np.maximum(1e-5, mel))
-        mel = np.clip((mel - ref_db + max_db) / max_db, 1e-8, 1)
-        mel = mel.T.astype(np.float32)
+        # mel_basis = librosa.filters.mel(require_sr, n_fft, n_mels)
+        # mel = np.dot(mel_basis, mag)  # (n_mels, t)
+        # mel = 20 * np.log10(np.maximum(1e-5, mel))
+        # mel = np.clip((mel - ref_db + max_db) / max_db, 1e-8, 1)
+        # mel = mel.T.astype(np.float32)
+        wav = load_wav(fpath)
+        mel = melspectrogram(wav)
```

Comment:
I think this function should have some augments right? You should pass our settings (sr, n_fft, n_mels) to here to align it with other stuffs.

Reply:
BTW, can you use the wave load by the librosa (why load the wave again at here?) If it's because some time shift, please change them here, other than re-load the wave